test/DynamoCoreWpfTests/DynamoTestUIBase.cs

```diff
@@ -132,10 +130,10 @@ protected virtual DynamoModel.IStartConfiguration CreateStartConfiguration(IPath
         [TearDown]
         public void Exit()
         {
+          
+            Dispatcher.CurrentDispatcher.Hooks.OperationPosted -= Hooks_OperationPosted;
```

Comment:
can you try to the handler cleanup after DoEventsLoop ? or did you try and it broke everything ?